### PR TITLE
KZOO-252: fixed call not dropping after 3 invalid attempts

### PR DESCRIPTION
**Pull request by RefathWadood512**

### CODE REVIEW

**Review by icehess — CHANGES_REQUESTED** [review]
You need to format the code, check the CI log.

**Comment by RefathWadood512** [comment]
Fixed whitespace and ran` make ci-codechecks `which passed.

File: src/module/cf_voicemail.erl [hunk 1]
```diff
@@ -1401,13 +1401,14 @@ delete_message(Message, AccountId, BoxId, Call) ->
 -type message_menu_returns() :: {'ok', 'keep' | 'delete' | 'return' | 'replay' | 'prev' | 'next' | 'forward' | 'rewind' | 'fastforward' | 'callback'} |
                                 {'error', 'channel_hungup' | 'channel_unbridge' | kz_json:object()}.
 
--spec play_message_menu(mailbox(), kapps_call:call()) ->
+-spec play_message_menu(kapps_call_command:audio_macro_prompts(), mailbox(), kapps_call:call()) ->
           {'error', 'channel_hungup' | 'channel_unbridge' | kz_json:object()} |
           message_menu_returns().
-play_message_menu(Box, Call) ->
-    play_message_menu(message_menu(), Box, Call).
+play_message_menu(Promt, Box, Call) ->
```
Comment (jamesaimonetti) [inline src/module/cf_voicemail.erl]: Check spelling of Prompt

File: src/module/cf_voicemail.erl [hunk 2]
```diff
@@ -1448,9 +1449,12 @@ play_message_menu(Prompt, #mailbox{keys=#keys{replay=Replay
...
         {'error', _}=E -> E;
         _ ->
-            _ = kapps_call_command:b_prompt(<<"menu-invalid_entry">>, Call),
-            play_message_menu(Box, Call)
-    end.
+            play_message_menu(message_menu(), Box, Call, RetryLoop+1)
+    end;
+play_message_menu(_Promt, _Box, Call, RetryLoop) when RetryLoop >= ?MAX_MESSAGE_MENU_LOOPS ->
+    _ = kapps_call_command:b_prompt(<<"menu-invalid_entry">>, Call),
+    lager:debug("hanging up channel after ~p attempts to get a message action", [?MAX_MESSAGE_MENU_LOOPS]),
```
Comment (jamesaimonetti) [inline src/module/cf_voicemail.erl]: put this at info level

File: src/module/cf_voicemail.erl [hunk 3]
```diff
@@ -1448,9 +1449,12 @@ play_message_menu(Prompt, #mailbox{keys=#keys{replay=Replay
...
+    _ = kapps_call_command:b_prompt(<<"vm-goodbye">>, Call).
```
Comment (jamesaimonetti) [inline src/module/cf_voicemail.erl]: call kapps_call_command:hangup/1 next to force the hangup.

**Pull request merged**